sample-android-app/src/main/java/com/badoo/reaktive/sample/android/MainActivity.kt

```diff
@@ -24,11 +24,8 @@ class MainActivity : AppCompatActivity() {
         val textView = findViewById<TextView>(R.id.text)
 
         findViewById<Button>(R.id.button).setOnClickListener {
-            observableFromFunction<String> {
-                Thread.sleep(1000L)
```

Comment:
The idea was to simulate blocking operation on background thread here. Later I want to replace it with Kitties loading (like in JS sample).

Reply:
Reverted